integration-libs/digital-payments/src/checkout/dp-checkout.module.ts

```diff
@@ -1,18 +1,17 @@
-import { OccDigitalPaymentsAdapter } from './adapters/occ-digital-payments.adapter';
-import { OccDpDetailsNormalizer } from './adapters/occ-digital-payment-details.normalizer';
-import { OccDpRequestNormalizer } from './adapters/occ-digital-payment-request.normalizer';
-import { DpPaymentMethodModule } from './cms-components/dp-payment-method/dp-payment-method.module';
 import { NgModule } from '@angular/core';
-import { DigitalPaymentsAdapter } from './adapters/digital-payments.adapter';
 import { provideDefaultConfig } from '@spartacus/core';
 import { occDigitalPaymentsConfig } from './adapters/config/occ-digital-payments-endpoint.config';
 import {
   DP_DETAILS_NORMALIZER,
   DP_REQUEST_NORMALIZER,
 } from './adapters/converters';
+import { DigitalPaymentsAdapter } from './adapters/digital-payments.adapter';
+import { OccDpDetailsNormalizer } from './adapters/occ-digital-payment-details.normalizer';
+import { OccDpRequestNormalizer } from './adapters/occ-digital-payment-request.normalizer';
+import { OccDigitalPaymentsAdapter } from './adapters/occ-digital-payments.adapter';
 
 @NgModule({
-  imports: [DpPaymentMethodModule],
+  //imports: [DpPaymentMethodModule],
```

Comment:
I just temporally comment out this line to make checkout lib lazy loaded.

Reply:
Let's create a followup ticket to re-enable it